staker/staker.gno

```diff
@@ -99,13 +103,17 @@ func StakeToken(tokenId uint64) (string, string, string) { // poolPath, token0Am
 	hasInternal := poolHasInternal(poolPath)
 	hasExternal := poolHasExternal(poolPath)
 	if hasInternal == false && hasExternal == false {
-		panic(ufmt.Sprintf("[STAKER] staker.gno__StakeToken() || poolPath(%s) has no internal or external incentive(can not stake)", poolPath))
+		panic(addDetailToError(errNonIncentivizedPool,
+			ufmt.Sprintf("staker.gno__StakeToken() || poolPath(%s) has no internal or external incentive(can not stake)", poolPath),
+		))
 	}
 
 	// check tokenId has liquidity or not
 	liquidity := pn.PositionGetPositionLiquidity(tokenId) // *u256.Uint
 	if liquidity.Lte(u256.Zero()) {
-		panic(ufmt.Sprintf("[STAKER] staker.gno__StakeToken() || tokenId(%d) has no liquidity", tokenId))
+		panic(addDetailToError(errInvalidInput,
```

Comment:
If there is no liquidity, is it an input error?

Reply:
66dfdbc31355c842079738059affb0b6172cd923